packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -1179,6 +1179,7 @@ const schema: SchemaType<DbPost> = {
     type: Date,
     control: 'datetime',
     viewableBy: ['guests'],
+    // TODO: Feels like this should be more restrictive
     editableBy: ['members'],
```

Comment:
I straightforwardly think it should be. This currently lets anyone promote their post to the frontpage.